src/pages/contributors/alexander-brandes.adoc

```diff
@@ -0,0 +1,81 @@
+= Alexander Brandes
+:page-name: Alexander Brandes
+:page-linkedin: 
+:page-twitter: NotMyFault_OG
+:page-github: NotMyFault
+:page-email: 
+:page-image: avatar/alexander-brandes.jpg
+:page-pronouns: He/Him
+:page-location: Kassel, Germany
+:page-firstcommit: 2019
+:page-featured: true
+:page-intro: Alexander Brandes is an open-source contributor and maintainer and serves on the Jenkins governance board. He has made contributions to multiple aspects of the Jenkins project including Jenkins Core, multiple plugins, Weekly and Long Term Support releases, and the community.
```

Comment:
Are 'Weekly' and 'Long Term Support' proper nouns?

Reply:
They're not, you're right. I would keep LTS that way though.

Reply:
Thanks @liamfield and @lemeurherve, updated accordingly!